README.md

```diff
@@ -1,8 +1,10 @@
-Provides analytics and alerting for modqueues.
+# Modqueue Tools (modqueue-tools)
```

Comment:
If this was a standalone REEADME file on a repo, then I would want this here. However adding a heading here makes the app directory look weird, because Dev Platform shows its own heading.